test/onigumo_cli_test.exs

```diff
@@ -39,6 +44,12 @@ defmodule OnigumoCLITest do
       end
     end
 
+    for switches <- @invalid_switch_combinations do
```

Comment:
There is one more user story of providing both a positional argument and the help switch. If ran in the `./onigumo downloader -h` order, the `-h` switch is ignored.

This case is not tested and because it doesn’t work correctly, the test would fail. The expected result is a help message to be printed because of an invalid argument combination. The actual result is that the downloader is run.

Reply:
that should be already tested but it is strange that github does not mark as as outdated. I suppose it because the line of code you mark is not changed but in text is mentioned case of test which is not added in your changes.